src/style.scss

```diff
@@ -180,6 +180,12 @@
       line-height: var(--auro-text-body-height-default);
     }
   }
+
+  .price--empty {
+    @include auro_breakpoint--sm {
```

Comment:
Is the Figma design really to only have the lighter color font on larger breakpoints when empty? I would think they all would be? The Figma showing missing price design is not attached to the PR or the ticket and probably should be.